.github/workflows/self_hosted_e2e.yaml

```diff
@@ -23,23 +23,40 @@ jobs:
       ARCHIVE_OUTDIR: dist/archives
     strategy:
       matrix:
-        os: [ubuntu-latest]
-        target_arch: [amd64, arm64]
+        os: [ubuntu-latest, macos-latest]
+        target_arch: [amd64]
```

Comment:
is it possible to still have arm64 and exclude darwin arm64 combination if necessary ?

Reply:
We can... I'm not sure what the value of running the arm e2e tests on qemu on x86 is really? I mean, I guess there is value in making sure our arm binary works, but this isn't running on a real arm machine.

Reply:
oh ok

Reply:
Ok, so I've gone down the rabbit hole a bit. Zeroconf app discovery does not work in qemu-arm64
```
INFO[0000] mTLS is disabled. Skipping certificate request and tls validation  app_id=test2 instance=docker-desktop scope=dapr.runtime type=log ver=1.1.0                                                                                                                                                                                                                                                                                                                                                                                                                                    
Unknown QEMU_IFLA_INFO_KIND ipip                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
Unknown QEMU_IFLA_INFO_KIND ip6tnl                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
Unsupported setsockopt level=0 optname=42                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   
Unsupported setsockopt level=0 optname=42                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   
Unsupported setsockopt level=0 optname=42                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   
Unsupported setsockopt level=0 optname=42                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   
Unsupported setsockopt level=0 optname=42                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   
Unsupported setsockopt level=0 optname=42                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   
2021/04/05 17:40:18 [zeroconf] no suitable IPv4 interface: udp4: failed to join any of these interfaces: [{2 1500 eth0 02:50:00:00:00:01 up|broadcast|multicast} {6 1500 docker0 02:42:88:c0:ae:ec up|broadcast|multicast} {16 1500 vetha514d7f 52:cb:02:5a:de:c3 up|broadcast|multicast} {121 1500 vethd74f16d 66:15:fb:6d:1e:70 up|broadcast|multicast} {123 1500 veth113aea7 a2:e9:c6:7e:40:ec up|broadcast|multicast} {125 1500 veth74335f8 da:64:31:7a:d1:f3 up|broadcast|multicast}]                                                                                                  
Unsupported setsockopt level=41 optname=42                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Unsupported setsockopt level=41 optname=42                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Unsupported setsockopt level=41 optname=42                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Unsupported setsockopt level=41 optname=42                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Unsupported setsockopt level=41 optname=42                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
Unsupported setsockopt level=41 optname=42                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
2021/04/05 17:40:18 [zeroconf] no suitable IPv6 interface: udp6: failed to join any of these interfaces: [{2 1500 eth0 02:50:00:00:00:01 up|broadcast|multicast} {6 1500 docker0 02:42:88:c0:ae:ec up|broadcast|multicast} {16 1500 vetha514d7f 52:cb:02:5a:de:c3 up|broadcast|multicast} {121 1500 vethd74f16d 66:15:fb:6d:1e:70 up|broadcast|multicast} {123 1500 veth113aea7 a2:e9:c6:7e:40:ec up|broadcast|multicast} {125 1500 veth74335f8 da:64:31:7a:d1:f3 up|broadcast|multicast}]                                                                                                  
ERRO[0000] error from zeroconf register: No supported interface  app_id=test2 instance=docker-desktop scope=dapr.contrib type=log ver=1.1.0                                                                                                                                                                                                                                                                                                                                                                                                                                                 
ERRO[0000] failed to initialize name resolution resolver standalone: No supported interface  app_id=test2 instance=docker-desktop scope=dapr.runtime type=log ver=1.1.0                                                                                                                       
WARN[0000] failed to init name resolution: No supported interface  app_id=test2 instance=docker-desktop scope=dapr.runtime type=log ver=1.1.0    
```

Do we:
- try and find a workaround
- decide that qemu-arm64 isn't a supported CLI environment

Reply:
After talking with @pkedy, I don't think we actually intended to be doing a qemu-arm64 test here. I'm going to disable it for now. Going to bring up with the team setting up some BYO arm64 build machines for our github workflows.